measure_extinction/extdata.py

```diff
@@ -1018,14 +1044,14 @@ def fit_spex_ext(self):
         fit(func, waves, exts) : CompoundModel
             Fitting results
         """
-        # only get the SpeX data, and sort the curve from short to long wavelengths
+        # get the SpeX data, and sort the curve from short to long wavelengths
         (waves, exts, exts_unc) = self.get_fitdata(["SpeX_SXD", "SpeX_LXD"])
         indx = np.argsort(waves)
         waves = waves[indx].value
         exts = exts[indx]
         exts_unc = exts_unc[indx]
 
         # fit a powerlaw to the spectrum
-        func = PowerLaw1D() | AxAvToExv()
+        func = PowerLaw1D(fixed={"x_0": True}) | AxAvToExv(bounds={"Av": (0.0, 10.0)})
```

Comment:
It would be more flexible if the bounds were keyword inputs to this function.  Then they can be changed when called as needed.